src/blocks/page.scss

```diff
@@ -0,0 +1,6 @@
+.page {
+  margin: 0;
+  padding: $calendar-padding;
```

Comment:
The variable `$calendar-padding` is used here but not defined within this file. Ensure it is defined elsewhere in your project or imported if defined in another file.